kin_base/horizon.py

```diff
@@ -225,7 +224,7 @@ def __init__(self,
                 except aiohttp.ClientPayloadError:
                     # Retry if the connection dropped after we got the initial response
                     logger.debug('Resetting SSE connection for {} after timeout'.format(url))
-                    await asyncio.sleep(retry.total_seconds())
+                    await asyncio.sleep(retry)
```

Comment:
what's the fix here?

Reply:
We set ```retry``` to 0.1 at the start, so the bug was that if we never got an answer from the server, ```retry``` stayed a float, and didn't had the ```total_seconds()``` method. So now I do the conversion when I get the retry from the server, and not at the end

Reply:
👍